lib/absinthe/pipeline.ex

```diff
@@ -92,6 +92,7 @@ defmodule Absinthe.Pipeline do
       # Prepare for Execution
       Phase.Document.CascadeInvalid,
       Phase.Document.Flatten,
+      {Phase.Document.Complexity, options},
```

Comment:
Should this be optional phase? I think current implementation *must* go in this position.